linode/manager
# feat: [M3-7019] - Create add Nodebalancer to Firewall drawer

## Description 📝
Added the "Add NodeBalancer to Firewall Drawer"

## Major Changes 🔄
- Added "Add Nodebalancer to Firewall" Drawer
- Added a Notice to the top of the drawer - documentation link is still needed
- Updated the text at the top of the landing page for both Linodes and NodeBalancers
- Added toast notifications

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-08-30 at 2 42 19 PM](https://github.com/linode/manager/assets/139489745/e7ad14f3-b315-44df-8723-bff7efb29064) | <video src="https://github.com/linode/manager/assets/139489745/92ade8b7-fdfa-47ec-9cfd-df9038e62f66" /> |

## How to test 🧪
1. Navigate to the Firewalls landing page
3. Click on any of your active firewalls
4. Navigate to the Nodebalancers tab
5. Ensure when you add a Nodebalancer, a toast notification appears and the number next to 'Nodebalancer' also updates (don't worry about the table, thats my next ticket)
6. To test errors
   1. Create two test firewalls (firewall-test-1, firewall-test-2)
   2. Assign a nodebalancer and a linode to a firewall-test-1
   3. Try to assign the same linode and nodebalancer to firewal-test-2
   4. For both drawers an error notice should appear with the link to the nodebalancer/linode landing page
   5. Ensure that the toast notifications are correctly displayed
   6. Ensure that the errors are removed when closing and reopening the tab
